src/moonstone-samples/lib/HeaderSample.js

```diff
@@ -16,17 +16,17 @@ module.exports = kind({
 	classes: 'moon enyo-unselectable enyo-fit moon-header-sample',
 	components: [
 		{kind: Scroller, fit:true, components: [
-			{kind: Header, name: 'largeHeader', content: 'Large Header', titleAbove: '02', titleBelow: 'Sub Header', subTitleBelow: 'Sub-sub Header', components: [
+			{kind: Header, name: 'largeHeader', content: 'Largé Héader', titleAbove: '02', titleBelow: 'Sub Header', subTitleBelow: 'Sub-sub Header', components: [
```

Comment:
Hah! That slipped through. That was from my prior tests. Please remove if you can, otherwise I will tomorrow morning.